### PR TITLE
2.9.0

### PR DESCRIPTION
**Pull request by franmomu**

### CODE REVIEW

File: UPGRADE-2.x.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 UPGRADE 2.x
 ===========
 
-UPGRADE FROM 2.x to 2.x
+UPGRADE FROM 2.7 to 2.9
```
Comment (VincentLanglet) [inline UPGRADE-2.x.md]: 2.8 to 2.9 ? or 2.7 to 2.8 ?

Comment (franmomu) [reply UPGRADE-2.x.md]: When releasing 2.8 the `UPGRADE-2.x` file wasn't updated, I've fixed it.

Comment (VincentLanglet) [reply UPGRADE-2.x.md]: Cant we fix it by splitting the 2.7 to 2.9 to a 2.7 to 2.8 part and a 2.8 to 2.9 part instead ?

**Review by jordisala1991 — APPROVED** [review]

File: UPGRADE-2.x.md (hunk 1 above)
Comment (franmomu) [reply UPGRADE-2.x.md]: Yep, this is what I did

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**